.github/workflows/e2e-test-provider-example.yml

```diff
@@ -197,15 +227,23 @@ jobs:
               }
             }
           }
+
+          locals {
+            control_plane_count = 1
+            worker_count        = 1
+          }
```

Comment:
Is it intended for this test to only run on 1 worker + 1 control-plane node?
Seems to be easily changeable to a configurable input

Reply:
Not suggesting to change the default, just thought we might want to leave it configurable if that can be done easily (e.g. we havent hit the inputs limit on the workflow yet).

If not, just ignore.